R/make_instrument.R

```diff
@@ -13,18 +13,21 @@
 #' @export
 #'
 ## @examples
-make_instrument <- function(df, first_var, last_var, drop_which_when = FALSE) {
+make_instrument <- function(df, first_var, last_var, drop_which_when = FALSE, 
+                            record_id="record_id") {
```

Comment:
Need to document in @param
add @param record_id The name of the record_id variable (in case somebody changed it to record_number)